examples/src/main/java/io/grpc/examples/routeguide/RouteGuideServer.java

```diff
@@ -281,30 +283,31 @@ private Feature checkFeature(Point location) {
       // No feature was found, return an unnamed feature.
       return Feature.newBuilder().setName("").setLocation(location).build();
     }
+  }
 
-    /**
-     * Calculate the distance between two points using the "haversine" formula.
-     * This code was taken from http://www.movable-type.co.uk/scripts/latlong.html.
-     *
-     * @param start The starting point
-     * @param end The end point
-     * @return The distance between the points in meters
-     */
-    private static double calcDistance(Point start, Point end) {
-      double lat1 = RouteGuideUtil.getLatitude(start);
-      double lat2 = RouteGuideUtil.getLatitude(end);
-      double lon1 = RouteGuideUtil.getLongitude(start);
-      double lon2 = RouteGuideUtil.getLongitude(end);
-      int r = 6371000; // metres
-      double φ1 = toRadians(lat1);
-      double φ2 = toRadians(lat2);
-      double Δφ = toRadians(lat2 - lat1);
-      double Δλ = toRadians(lon2 - lon1);
-
-      double a = sin(Δφ / 2) * sin(Δφ / 2) + cos(φ1) * cos(φ2) * sin(Δλ / 2) * sin(Δλ / 2);
-      double c = 2 * atan2(sqrt(a), sqrt(1 - a));
-
-      return r * c;
-    }
+  /**
+   * Calculate the distance between two points using the "haversine" formula.
+   * This code was taken from http://www.movable-type.co.uk/scripts/latlong.html.
+   *
+   * @param start The starting point
+   * @param end The end point
+   * @return The distance between the points in meters
+   */
+  @VisibleForTesting
+  static double calcDistance(Point start, Point end) {
+    double lat1 = RouteGuideUtil.getLatitude(start);
+    double lat2 = RouteGuideUtil.getLatitude(end);
+    double lon1 = RouteGuideUtil.getLongitude(start);
+    double lon2 = RouteGuideUtil.getLongitude(end);
+    int r = 6371000; // metres
+    double φ1 = toRadians(lat1);
```

Comment:
as cool as this is, the google internal linter complains about it, and it doesn't read fast.  (everything else in this file is english, why switch to greek?)

Consider switching this to spelled out names.